gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_0_0/schema.yml

```diff
@@ -37,13 +34,10 @@ databaseChangeLog:
         - createTable:
             tableName: ${gravitee_prefix}events_latest_properties
             columns:
-              - column: {name: event_id, type: nvarchar(64), constraints: { nullable: false } }
-              - column: {name: property_key, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: event_id, type: nvarchar(64), constraints: { nullable: false, primaryKey: true, primaryKeyName: pk_events_latest_properties } }
+              - column: {name: property_key, type: nvarchar(64), constraints: { nullable: false, primaryKey: true, primaryKeyName: pk_events_latest_properties } }
```

Comment:
The usage of the dynamic values ( `${gravitee_prefix}` ) looks not supported in the column constraint primaryKeyName.
I chose to simply not use it. This is something we can discuss I don't know if it's a blocker 🤷🏼 Another possibility is to not set it manually.

Reply:
For me it should be ok  🤷

Reply:
Agree, it seems to be an ok tradeoff